paddle/fluid/operators/CMakeLists.txt

```diff
@@ -193,6 +193,7 @@ if(WITH_DISTRIBUTE)
     set_source_files_properties(send_vars_op.cc PROPERTIES COMPILE_FLAGS ${DISTRIBUTE_COMPILE_FLAGS})
     op_library(send_barrier_op DEPS ${DISTRIBUTE_DEPS})
     set_source_files_properties(send_barrier_op.cc PROPERTIES COMPILE_FLAGS ${DISTRIBUTE_COMPILE_FLAGS})
+    set_source_files_properties(send_recv_op_test.cc PROPERTIES COMPILE_FLAGS ${DISTRIBUTE_COMPILE_FLAGS})
```

Comment:
Please remove the use of `set_source_files_properties` as discussed in https://github.com/PaddlePaddle/Paddle/issues/9612

Reply:
Because of grpc implementation, we have to use this when linking with grpc, and this can not be present at generic.cmake.